README.md

```diff
@@ -1 +1,25 @@
-# knotx-adapter-archetype
\ No newline at end of file
+![Knot.x logo](https://avatars2.githubusercontent.com/u/24564424?v=3&s=400)
+
+# Knot.x Extension Maven Archetype
+
+Use this archetype to generate a custon _Knot.x_ extension.
```

Comment:
Actually it looks that this archetype is for building `Adapter` extension.
It will be separate extension for new Knot in the future?